web/main.js

```diff
@@ -249,6 +253,18 @@ let preventdefault = e => {
 	e.stopPropagation()
 }
 
+let joining = () => {
+	document.getElementById("magiccode").value = location.hash.substring(1);
+		document.getElementById("dial").value = "JOIN WORMHOLE";
+		connect();
```

Comment:
i think there's a couple of spare tabs in these lines :)

Reply:
True, this used to be an if statement ;) Removing!